processing/src/main/java/org/apache/druid/query/filter/IntervalDimFilter.java

```diff
@@ -59,10 +63,21 @@ public IntervalDimFilter(
     this.dimension = dimension;
     this.intervals = Collections.unmodifiableList(JodaUtils.condenseIntervals(intervals));
     this.extractionFn = extractionFn;
+    this.filterTuning = filterTuning;
     this.intervalLongs = makeIntervalLongs();
     this.convertedFilter = new OrDimFilter(makeBoundDimFilters());
   }
 
+  @VisibleForTesting
+  public IntervalDimFilter(
```

Comment:
In general I assume you left the old constructors in to avoid a bunch of test churn. I think that's fine but could you please double check that they aren't being used in production code? (You got the one in `InDimFilter.optimize` that I was thinking of when I wrote this, but there are potentially others)

Reply:
there were a few being created in calcite stuff, adjusted them to set null for tuning.